app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -273,7 +270,15 @@ class HomePresenter(
         return if (usernameExists) {
             listOf(feedbackBanner)
         } else {
-            listOf(usernameBanner, feedbackBanner)
+            listOf(
+                Banner(
```

Comment:
maybe extract this Banner creation to a local variable and then rewrite it as

listOf(mainBanner, feedbackBanner)?
i cant understand what banner this is.

Reply:
hmmm maybe you are right